states/_dbmgmt.sls

```diff
@@ -8,6 +8,15 @@
 include:
   - mysql._salt
 
+{% if salt['grains.get']('os_family') in ['RedHat', 'Suse' ] %}
```

Comment:
Is this RedHat/ Suse specific? AFAIR when you install mysql/ mariadb on Debian/ RedHat root doesn't have a password. Then you can use https://github.com/bechtoldt/saltstack-mysql-formula/blob/master/states/_dbmgmt.sls#L38 to set a new one. But I might be wrong on this.